cmd/vulcan-exposed-ssh/main.go

```diff
@@ -176,10 +184,12 @@ func (r *runner) gradeVuln() ([]report.Vulnerability, error) {
 			// NOTE: this doesn't cover different types of errors that ssh_scan can produce
 			continue
 		}
-		vulnDb := map[string]report.Vulnerability{}
 		// Exposed SSH
-		exposedSSHVuln.Details += fmt.Sprintf("* Exposed SSH Port in %v\n", i.Port)
-		vulnDb["exposedSSHVuln"] = exposedSSHVuln
+		v := exposedSSHVuln
+		v.Details += fmt.Sprintf("* Exposed SSH Port in %v\n", i.Port)
+		v.AffectedResource = fmt.Sprintf("%d/%s", i.Port, "ssh")
+		v.Fingerprint = helpers.ComputeFingerprint(v.Summary, v.Score)
```

Comment:
If the summary changes, the entire issue will change so it's not useful for the fingerprint. I'd rather use or empty args, or the `ServerBanner` value. WDYT?

Reply:
Yes, good point. I'll remove the args from the fingerprint call.